dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryDirectory.java

```diff
@@ -426,7 +435,7 @@ private URL mergeUrl(URL providerUrl) {
         this.overrideDirectoryUrl = this.overrideDirectoryUrl.addParametersIfAbsent(providerUrl.getParameters()); // Merge the provider side parameters
 
         if ((providerUrl.getPath() == null || providerUrl.getPath()
-                .length() == 0) && "dubbo".equals(providerUrl.getProtocol())) { // Compatible version 1.0
+                .length() == 0) && Constants.DOBBO_PROTOCOL.equals(providerUrl.getProtocol())) { // Compatible version 1.0

```

Comment:
The pr #3491  which has been merged into the main branch has changed this constant name. 
Would you please update the latest code and change it here?

Reply:
@lixiaojiee done